studio/components/layouts/ProjectLayout/PausingState.tsx

```diff
@@ -1,11 +1,41 @@
-import { Project } from 'types'
+import { useQueryClient } from '@tanstack/react-query'
+import { useParams } from 'common'
 import { Badge, IconCircle, IconLoader } from 'ui'
 
+import { useProjectStatusQuery } from 'data/projects/project-status-query'
+import { invalidateProjectsQuery } from 'data/projects/projects-query'
+import { PROJECT_STATUS } from 'lib/constants'
+import { Project } from 'types'
+import { useEffect, useState } from 'react'
+
 export interface PausingStateProps {
   project: Project
 }
 
 const PausingState = ({ project }: PausingStateProps) => {
+  const { ref } = useParams()
+  const queryClient = useQueryClient()
+  const [startPolling, setStartPolling] = useState(false)
+
+  useProjectStatusQuery(
+    { projectRef: ref },
+    {
+      enabled: startPolling,
+      refetchInterval: (res) => {
+        return res?.status === PROJECT_STATUS.INACTIVE ? false : 2000
+      },
+      onSuccess: async (res) => {
+        if (res.status === PROJECT_STATUS.INACTIVE) {
+          await invalidateProjectsQuery(queryClient)
```

Comment:
All the requests seem to be working now, but the pausing state remains... Should this be `setProjectStatus` or invalidate project **detail** query instead of `invalidateProjectsQuery`?

Reply:
ohhh yes you're right!

Reply:
okay just pushed again 🙏  - I also update BuildingState to do the same too, was wondering why locally the page doesnt automatically update after the project is active_healthy